hugo/content/Configuration/compatibility.md

```diff
@@ -74,11 +74,17 @@ dependency between the two projects at a specific version level.
 ## Operating Systems
 
 The Operator is developed on both Centos 7 and RHEL 7 operating systems.  The
-underlying containers are designed to use either Centos 7 or RHEL 7 as the base
-container image.
+underlying containers are designed to use either Centos 7 or Red Hat UBI 7 as 
+the base container image.
 
 Other Linux variants are possible but are not supported at this time.
 
+Also, please note that as of version 4.2.2 of the PostgreSQL Operator, Red Hat Univeral Base
+Image (UBI) 7 has replaced RHEL 7 as the base container image for the various PostgreSQL Operator
+containers.  For more information on Red Hat UBI, please see the following link:
+
+https://www.redhat.com/en/blog/introducing-red-hat-universal-base-image
```

Comment:
How does it look with a hyperlink instead of a sentence followed by a URL?

```suggestion
Also, please note that as of version 4.2.2 of the PostgreSQL Operator, Red Hat [Univeral Base
Image][UBI] (UBI) 7 has replaced RHEL 7 as the base container image for the various PostgreSQL Operator
containers.

[UBI]: https://www.redhat.com/en/blog/introducing-red-hat-universal-base-image
```